pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/AggregationFunctionType.java

```diff
@@ -254,12 +254,12 @@ public enum AggregationFunctionType {
 
   // argMin and argMax
   // TODO: argmin/argmax syntax not conformed with Calcite. we need to migrate.
-  ARGMIN("argMin"),
-  ARGMAX("argMax"),
-  PARENTARGMIN(CommonConstants.RewriterConstants.PARENT_AGGREGATION_NAME_PREFIX + ARGMIN.getName()),
-  PARENTARGMAX(CommonConstants.RewriterConstants.PARENT_AGGREGATION_NAME_PREFIX + ARGMAX.getName()),
-  CHILDARGMIN(CommonConstants.RewriterConstants.CHILD_AGGREGATION_NAME_PREFIX + ARGMIN.getName()),
-  CHILDARGMAX(CommonConstants.RewriterConstants.CHILD_AGGREGATION_NAME_PREFIX + ARGMAX.getName()),
+  EXPRMIN("exprMin"),
```

Comment:
You may need to update here for making v2 query engine work

Reply:
Updated, PTAL, thanks!